### PR TITLE
iOS support

### PR DESCRIPTION
**Pull request by ospfranco**
One small correction on the `body.h` header file where clang fails with an implicit cast and added another flag to prevent searching for CA files which on sandboxed iOS apps is not possible and also not necessary since using Secure Transport already uses the OS layer for TLS verification.

Partially takes care of #715 I'm still playing around but I will provide complete instructions to build it later.

### CODE REVIEW

File: include/cpr/body.h [hunk 1]
```diff
@@ -31,7 +31,7 @@ class Body : public StringHolder<Body> {
         std::ifstream is(file.filepath, std::ifstream::binary);
         if (is) {
             is.seekg(0, is.end);
-            int length = is.tellg();
+            int length = (int)is.tellg();
```
Comment (KingKili) [inline include/cpr/body.h:34]: Please don't use C-style casts. Use a `static_cast` instead

**Review by KingKili — CHANGES_REQUESTED** [review]
Thanks for your work. What is the reason for the [.DS_Store](.DS_Store) file? If it is not required please also remove it.

**Comment by ospfranco** [comment]
Don't you want to add .DS_store to gitignore? I guess people using macs also contribute every once in a while :)

**Comment by KingKili** [comment]
> Don't you want to add .DS_store to gitignore? I guess people using macs also contribute every once in a while :)

Great Idea!

**Review by KingKili — APPROVED** [review]

**Review by KingKili — APPROVED** [review]

**Comment by KingKili** [comment]
Thanks for your good work. I will merge it once the CI is run.

**Review by COM8 — APPROVED** [review]

**Pull request merged**